celavi/__main__.py

```diff
@@ -204,6 +207,11 @@
                     turbine_data_filename,
                     states_to_filter)
 
+print('State filtering completed in %d s' % np.round(time.time() - time0, 1),
+        flush=True)
+
+time0 = time.time()
```

Comment:
Is it necessary to time the filtering operation? When I run it on my machine it takes very little time, < 1 second.